quesma/util/utils.go

```diff
@@ -852,6 +852,9 @@ func replaceNonAlphabetic(str string) string {
 }
 
 func FieldToColumnEncoder(field string) string {
+	if len(field) == 0 {
+		return field
+	}
```

Comment:
I think this `FieldToColumnEncoder` deserves just a few, very simple unit tests just in case anyone accidentally thinks that "oh, let's change this" after few months from now 🙃 
Also, a table style test would nicely illustrate how the algorithm and expected output looks like in general

Reply:
Tests were added in previous PR https://github.com/QuesmaOrg/quesma/blob/main/quesma/util/utils_test.go#L825